src/components/control-panel-pages/employees/employees-list/employees-list.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./employees-list.css";
+import endpointUrl from "../../../../config";
+
+const handleDelete = _id => {
+  fetch(`${endpointUrl}/delete-user`, {
```

Comment:
You need to include the id of the user on the end of URL, not in the body

eg: `${endpointUrl}/delete-user/${_id}`